_docs/examples/macos.md

```diff
@@ -11,3 +11,44 @@ If you run your Frida tool via the GUI with your local user (e.g. from Terminal.
 You may also need to disable [System Integrity Protection](https://support.apple.com/en-us/HT204899).
 
 _Please click "Improve this page" above and add an example. Thanks!_
+
+{% highlight py %}
+
+import frida
+import sys
+
+def on_message(message, data):
+    print("[%s] => %s" % (message, data))
+
+def main(target_process):
+    session = frida.attach(target_process)
+
+    script = session.create_script("""
+	// Get a reference to the openURL selector
+  var appWillFinishLaunch = ObjC.classes.NSApplicationDelegate["- applicationWillFinishLaunching:"];
+
+  // Intercept the method
+  Interceptor.attach(appWillFinishLaunch.implementation, {
+    onEnter: function(args) {
+      // As this is an ObjectiveC method, the arguments are as follows:
+      // 0. 'self'
+      // 1. The selector (applicationWillFinishLaunching:)
+      // 2. The first argument to the this selector
+      var myNotification = new ObjC.Object(args[2]);
+
+      // Convert it to a JS string
+      var myJSNotification = myNotification.absoluteString().toString();
+      // Log it
```

Comment:
This comment seems redundant, so perhaps it's more readable to merge the two comments into: `Convert it to a JS string and log it`.